workers/generic-worker/README.md

```diff
@@ -1,782 +1,12 @@
-*or* if you do not currently have a mozillians account but would like to create
-one.
-
-* Sign up for a [Mozillians account](https://mozillians.org/en-US/) (if you do not already have one)
-* Request membership of the [taskcluster-contributors](https://mozillians.org/en-US/group/taskcluster-contributors/) mozillians group
-
-## Option 2
-
-This method is for those who wish not to create a mozillians account, but
-already authenticate into taskcluster via some other means, or have a
-mozillians account but it is registered to a different email address than the
-one they use to log into Taskcluster with (e.g. via LDAP integration).
-
-* Request the scope `assume:project:taskcluster:generic-worker-tester` to be
-  granted to you via a [bugzilla
-  request](https://bugzilla.mozilla.org/enter_bug.cgi?product=Taskcluster&component=Service%20Request),
-  including your [currently active `ClientId`](https://tools.taskcluster.net/credentials/)
-  in the bug description. From the ClientId, we will be able to work out which role to assign the scope
-  to, in order that you acquire the scope with the client you log into Taskcluster tools site with.
-
-Once you have been granted the above scope:
-
-* If you are signed into tools.taskcluster.net already, **sign out**
-* Sign into [tools.taskcluster.net](https://tools.taskcluster.net/) using either your new Mozillians account, _or_ your LDAP account **if it uses the same email address as your Mozillians account**
-* Check that a role or client of yours appears in [this list](https://tools.taskcluster.net/auth/scopes/assume%3Aproject%3Ataskcluster%3Ageneric-worker-tester)
-* Create a permanent client (taskcluster credentials) for yourself in the [Client Manager](https://tools.taskcluster.net/auth/clients/) granting it the single scope `assume:project:taskcluster:generic-worker-tester`
-
-# Set up your env
-
-* Generate an ed25519 key pair with `generic-worker new-ed25519-keypair --file <file>` where `file` is where you want the generated ed25519 private key to be written to
-* Create a generic worker configuration file somewhere, with the following content:
-
-```
-{
-    "accessToken":                "<access token of your permanent credentials>",
-    "certificate":                "",
-    "clientId":                   "<client ID of your permanent credentials>",
-    "ed25519SigningKeyLocation":  "<file location you wrote ed25519 private key to>",
-    "livelogSecret":              "<anything you like>",
-    "provisionerId":              "test-provisioner",
-    "publicIP":                   "<ideally an IP address of one of your network interfaces>",
-    "workerGroup":                "test-worker-group",
-    "workerId":                   "test-worker-id",
-    "workerType":                 "<a unique name that only you will use for your test worker(s)>"
-}
-```
-
-To see a full description of all the config options available to you, run `generic-worker --help`:
```

Comment:
I also find this extremely helpful, since people typically don't realise the worker comes with fanstastic command line docs. This section used to be automatically rebuilt on release, so that it was always the latest version of the docs. We could integrate that approach in the monorepo too, so that it gets updated on release too, or even every commit. I would really like to keep it too.

It feels like the correct place for this is the mono repo README.md files rather than the docs site, so I think it was in the right place. I see the taskcluster online docs as user-facing, whereas the github repo is for people that build/deploy/develop taskcluster, and therefore in my opinion this is the perfect place for showing command line options of generic-worker.

Reply:
> I see the taskcluster online docs as user-facing, whereas the github repo is for people that build/deploy/develop taskcluster, and therefore in my opinion this is the perfect place for showing command line options of generic-worker.

Close -- deployers should be looking at the [online docs](https://docs.taskcluster.net/docs/manual/deploying), too.  The repo is only for developers.  So ultimately, this information should be in the online docs in some form.  For the moment, though, I can leave it in the README.